DoSomething/quasar
# Replicate database schemas for local dev

#### What's this PR do?
Replicates database schemas in local environment

#### Where should the reviewer start?
Look at provision.sh and Makefile

#### How should this be manually tested?
Try running:
`vagrant reload --provision`
OR, to be extra safe (takes longer):
```
vagrant destroy
vagrant up
```

Then check that the dbs and tables are there:
```
mysql -uroot -ppassword --host 127.0.0.1 --port 6603
show databases;
use quasar;
show tables;
```

or use your preferred client.

#### Any background context you want to provide?
The local db is only for testing and should be completely idempotent (we don't save any data). We might want to add some seed data as a next step. 

#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
